lib/sqlite_manager.rb

```diff
@@ -1,29 +1,25 @@
-require 'sqlite3'
-
 require_relative './row_transformer'
 
-# Client for creating and updating local sqlite3 database
+# Manages fetching and parsing records
 class SQLITEClient
```

Comment:
I'm not sure about the best approach but I think that either refactoring this class to be like `RowParser` or moving the transformer logic to the handler module would make sense.